.github/workflows/security-scanning.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# The default values used in the docker build commands are the root
+# directory '.' and the dockerfile name of 'Dockerfile'. If there is 
+# a need to change these do so in your local workflow template (this file) and
+# change them there. HINT: Look at the bottom of this file.
+
+# This workflow checks out code, builds an image, performs a container image
+# vulnerability scan with Anchore's Grype tool, and generates an
+# SBOM via Anchore's Syft tool
+
+# For more information on Anchore's container image scanning tool Grype, see
+# https://github.com/anchore/grype
+
+# For more information about the Anchore SBOM tool, Syft, see
+# https://github.com/anchore/syft
+
+name: ConsoleDot Platform Security Scan
+
+on:
+  push:
+    branches: [ "main", "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main", "master" ]
+
+jobs:
+  PlatSec-Security-Workflow:
+    uses: RedHatInsights/platform-security-gh-workflow/.github/workflows/platsec-security-scan-reusable-workflow.yml@master
```

Comment:
@casey-williams-rh are you certain to use `master` branch? Is there a more secure way?

Reply:
@vkrizan , We use the master branch that way we can push updates we need without having everyone or us update there workflow file. We have tests in place to ensure no braking changes are introduced when updates are merged into master.

Reply:
@casey-williams-rh I understand the the concept. However, I do not see this being a secure choice, pipeline security wise. What I think would be a better choice, is to use versioned workflows (with standard GitHub Actions format for the repo) and have a Dependabot create an update whenever a new version is provided. Otherwise, whomever have write access to the `platform-security-gh-workflow` repo would have access to all workflow runs.